Classes/PostProcessor/PublishStatusPostProcessor.php

```diff
@@ -29,14 +28,13 @@ class PublishStatusPostProcessor extends AbstractPostProcessor
      * @param array $response
      * @return array
      */
-    public function process($response)
+    public function process($response): array
```

Comment:
Why not: process(array $response)?

Reply:
Because the PostProcessorInterface has "mixed" for this property.

Reply:
In that case change Parameter in Annotation to mixed, too.